Index.html

```diff
@@ -0,0 +1,119 @@
+        <ul>
+            <li><a class="nav-link" href="Introduction">Introduction</a></li>
+            <li><a class="nav-link" href="What_you_should_know">What you should know</a></li>
+            <li><a class="nav-link" href="While_Statement">While Statement</a></li>
+            <li><a class="nav-link" href="Hello_World">Hello World</a></li>
+            <li><a class="nav-link" href="Variables">Variables</a></li>
+
+        </ul>
+    </nav>
+
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<main id="main-doc">Technical Documentation
+    <br>
+    <section class="main-section" id="Introduction">
+        <div>
+            <code>Introduction</code>
+        </div>
+
+        <br>
+        <header>Introduction</header>
+        <p>
+        <ul>
+            <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its
+                Document Object Model (DOM). For example, client-side extensions allow an application to place elements
+                on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+            <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on
+                a server. For example, server-side extensions allow an application to communicate with a database,
+                provide continuity of information from one invocation to another of the application, or perform file
+                manipulations on a server.</li>
+        </ul>
+        </p>
+        <br>
+        <div>
+            <section class="main-section" id="What_you_should_know">
+                <header>What you should know</header>
+                <div>
+                    <code>What you should know</code>
+                </div>
+
+                <p>
+                <ul>
+                    <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+                    <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+                </ul>
+                </p>
+
+                <br>
+                <div>
+                    <section class="main-section" id="While_Statement">
+                        <header>While Statement</header>
+                        <div>
+                            <code>While Statement</code>
+                        </div>
+
+                        <p>
+                            <li>After the first pass: n = 1 and x = 1</li>
+                            <li>After the second pass: n = 2 and x = 3</li>
+                        </p>
+
+                        <div>
+                            <section class="main-section" id="Hello_World">
+                                <header>Hello World</header>
+                                <div>
+                                    <code>Hello World</code>
```

Comment:
Code tags are used to define a piece of computer code, kindly use appropriate tags for the "Hello World" heading.